MockKitUI/View Controllers/MockCGMManagerSettingsViewController.swift

```diff
@@ -468,9 +521,36 @@ extension MockCGMManagerSettingsViewController: PercentageTextFieldTableViewCont
                 cgmManager.dataSource.effects.randomErrorChance = chance.clamped(to: 0...100)
             }
             tableView.reloadRows(at: [indexPath], with: .automatic)
+        case [Section.progress.rawValue, ProgressRow.percentComplete.rawValue]:
```

Comment:
[nit] This is a pretty non-standard way of matching section and row for the Loop projects.

Reply:
was following the existing pattern in this file, but will update to the standard pattern.